daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
Isn't the sandbox id is the same in both cases (w/ hooks and w/o hooks). Also I think the string returned by `Sandbox.Key()` which is coming from `osl.Generatekey(containerID string)` is the same in both cases, given it only takes the container id as input.

If both the above hold true, maybe we do not need to move this call.